sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/models/CreateKeyOptions.java

```diff
@@ -11,7 +11,7 @@
 import java.util.List;
 import java.util.Map;
 
-public class KeyCreateOptions {
+public class CreateKeyOptions {
```

Comment:
In subclasses of this class, you need to override all setter methods to ensure you return the base type and not this super type.

Reply:
Already done, only had one mistake corrected above.